jwst/pipeline/calwebb_spec2.py

```diff
@@ -39,7 +39,8 @@ class Spec2Pipeline(Pipeline):
     """
 
     spec = """
-        save_bsub = boolean(default=False) # Save background-subracted science
+        save_bsub = boolean(default=False)        # Save background-subracted science
+        fail_on_exception = boolean(default=True) # Fail if any product fails.
```

Comment:
You may want to add this `fail_on_exception` spec member to the default .cfg file for this pipeline.  =)

Reply:
Otherwise looks good.